app/data/session-data-defaults.js

```diff
@@ -475,9 +508,12 @@ module.exports = {
             name: "Accommodation",
             providerName: "XYZ Provider Name",
             assignedCaseworker: "Jenny Thompson",
-            actionPlanSubmitted: false,
-            actionPlanApproved: false,
-            assigned: false,
+            actionPlanSubmitted: true,
```

Comment:
Some of these changes to the fields which are also used by Record don't quite make sense from a Record point of view - e.g. a submitted or approved action plan should have goals or sessions. And an approved action plan should have an `actionPlanApprovedAt` timestamp.

I think you could either fix up the data here to make it make sense for Record, or you could just introduce namespaced versions of these things (i.e. `monitor.actionPlanSubmitted` etc) since we don't have any real need for Monitor and Record to be in sync at the moment and if we do later then we can sort this out.

I don't think you should start fixing up all of the data here just to make Record work, since at the moment we don't need there to be a link between the two

Reply:
Have just sent you a slack about this, but maybe it's better to leave the conversation here too:

Are the fields I should have left as is (and should namespace): `assigned`, `actionPlanApproved` and `actionPlanSubmitted`? (Basically anywhere I've changed an existing value, rather than added a new one?)